README.md

```diff
@@ -108,6 +108,69 @@ HTML styled "just right".
 Jekyll will print a local URL where the site can be viewed (usually,
 [http://0.0.0.0:4000/](http://0.0.0.0:4000/)).
 
+### Testing using Docker environment 
+
+#### Build environment
+A containerized development environment can be built using the local
+Dockerfile.\
```

Comment:
I see you're using a lot of backslashes here in the README to add arbitrary line breaks. Micro-managing the flow like that is a very different style of writing from our usual Markdown. I recommend just sticking to regular paragraph flow, and let the browser handle line wrapping automatically for long lines. If a sentence needs to stand out on its own, rather than just using a line break, just put a blank line between to make a new new paragraph. My understanding is that separate paragraphs are also better for some screen readers for people with low vision, as well, but I could be wrong about that. At the very least, that would be more consistent with our existing style, and a lot easier to maintain.

```suggestion
Dockerfile.

```

Reply:
Paragraph flow just looks very cluttered to me. 
I'm also unaware of any screen reader benefits but I'll switch these over anyway.